framework/wazuh/core/config/models/ssl_config.py

```diff
@@ -88,6 +91,38 @@ def validate_ssl_files(cls, path: str, info: ValidationInfo) -> str:
         
         return path
 
+    @field_validator('certificate_authorities')
+    @classmethod
+    def validate_cs_files(cls, paths: List[str], info: ValidationInfo) -> List[str]:
+        """Validate that the SSL certificate authorities files exist.
+
+        Parameters
+        ----------
+        paths : List[str]
+            Paths to the SSL certificate authorities.
+        info : ValidationInfo
+            Validation context information.
+
+        Raises
+        ------
+        ValueError
+            Invalid SSL file path.
+
+        Returns
+        ------
+        List[str]
+            SSL Certificate Authorities paths.
+        """
+        if info.data['use_ssl']:
+            for path in paths:
+                if path == '':
+                    raise ValueError(f'{info.field_name}: missing certificate file')
+
+                if not os.path.isfile(path):
+                    raise ValueError(f"{info.field_name}: the file '{path}' does not exist")
```

Comment:
Let's create an auxiliary private method for this code section since it is duplicated from the above method.

Reply:
Changed in efc092e8806ec38cc7ac3cf7e829d97872bd4dae.